addons/account/models/partner.py

```diff
@@ -646,7 +646,13 @@ def action_view_partner_invoices(self):
             ('move_type', 'in', ('out_invoice', 'out_refund')),
             ('partner_id', 'in', all_child.ids)
         ]
-        action['context'] = {'default_move_type': 'out_invoice', 'move_type': 'out_invoice', 'journal_type': 'sale', 'search_default_unpaid': 1}
+        action['context'] = {
+            'default_move_type': 'out_invoice',
+            'move_type': 'out_invoice',
+            'journal_type': 'sale',
+            'search_default_unpaid': 1,
+            'search_default_partner_id': all_child.ids,
```

Comment:
Even if we include all ids, if we don't have active_test here either won't they just be filtered out when the actual search happens?

Also maybe we can remove the forced domain if it overlaps? Otherwise removing the filter might have no effect

Reply:
Here active_test is for the archive partner, not the target records(cards). We handle it if the partner is archived and the target record is not. Even the counts on the stat button only consider the active records

Reply:
Makes sense, ok then